google-apps-feedback-script.js

```diff
@@ -0,0 +1,76 @@
+// The Google Apps Script that you install in Google sheet to make it a feedback endpoint
+// based on:
+// https://mashe.hawksey.info/2014/07/google-sheets-as-a-database-insert-with-apps-script-using-postget-methods-with-ajax-example/
+
+//  1. Enter sheet name where data is to be written below
+        var SHEET_NAME = "Sheet1";
+
+//  2. Run > setup
+//
+//  3. Publish > Deploy as web app
+//    - enter Project Version name and click 'Save New Version'
+//    - set security level and enable service (most likely execute as 'me' and access 'anyone, even anonymously)
+//
+//  4. Copy the 'Current web app URL' and post this in your form/script action
+//
+//  5. Insert column names on your destination sheet matching the parameter names of the data you are passing in (exactly matching case)
+
+var SCRIPT_PROP = PropertiesService.getScriptProperties(); // new property service
+
+// If you don't want to expose either GET or POST methods you can comment out the appropriate function
+function doGet(e){
+  return handleResponse(e);
+}
+
+function doPost(e){
+  return handleResponse(e);
+}
+
+function handleResponse(e) {
+  // shortly after my original solution Google announced the LockService[1]
+  // this prevents concurrent access overwritting data
+  // [1] http://googleappsdeveloper.blogspot.co.uk/2011/10/concurrency-and-google-apps-script.html
+  // we want a public lock, one that locks for all invocations
+  var lock = LockService.getPublicLock();
+  lock.waitLock(30000);  // wait 30 seconds before conceding defeat.
+
+  try {
+    MailApp.sendEmail("person@domain.org", "[GetHelpLex feedback]", JSON.stringify(e.parameters));
```

Comment:
Yah, it can be a [comma separated list](https://developers.google.com/apps-script/reference/mail/mail-app#sendEmail%28String,String,String%29) of addresses as well